engine/src/main/java/org/terasology/engine/subsystem/config/BindsSubsystem.java

```diff
@@ -178,7 +178,7 @@ private void addDefaultBindings(SimpleUri bindUri, Class<?> event, BindsConfigur
 
     private List<Input> fetchDefaultBindings(Class<?> event, BindsConfiguration config) {
         List<Input> defaultInputs = Lists.newArrayList();
-        Collection<Input> values = config.values();
+        Collection<Input> values = config.getBoundInputs();
```

Comment:
`values` (generic map term) --> `boundInputs` (the actual content of the collection).